test/debtToken.js

```diff
@@ -245,7 +243,7 @@ contract('DebtToken', function(accounts){
         });
         
         it('Should confirm loanValue does not increase after refundLoan',function(done){
-          var time = deployment_config._loanCycle*2*deployment_config._dayLength*1000;
+          var time = deployment_config._loanCycle*2*deployment_config._dayLength*2000;
```

Comment:
could you explain the increase to `2000` from `1000`?

Reply:
To be reverted, some testing leftovers.